pkg/controller/job/OWNERS

```diff
@@ -2,7 +2,6 @@
 
 approvers:
   - sig-apps-approvers
-  - alculquicondor
```

Comment:
we have emeritus , you deserve it

Reply:
```
emeritus_approvers:
  - alculquicondor
 ```

Reply:
I forgot about that, thanks :)

Just here is enough, we don't need to repeat in the test folder.